packages/backend/src/backend_factory.test.ts

```diff
@@ -150,11 +150,17 @@ void describe('Backend', () => {
     rootStackTemplate.resourceCountIs('Custom::AmplifyBranchLinkerResource', 0);
   });
 
-  void describe('getStack', () => {
+  void describe('createStack', () => {
     void it('returns nested stack', () => {
       const backend = new BackendFactory({}, rootStack);
-      const testStack = backend.getStack('testStack');
+      const testStack = backend.createStack('testStack');
       assert.strictEqual(rootStack.node.findChild('testStack'), testStack);
     });
+
+    void it('throws if stack has already been created with specified name', () => {
+      const backend = new BackendFactory({}, rootStack);
+      backend.createStack('testStack');
+      assert.throws(() => backend.createStack('testStack'), 'test message');
```

Comment:
`'test message'`?

Reply:
Fixed. Needed to put it in {message: 'thing'}. Slightly concerning that doesn't fail the assertion though

Reply:
This pattern is maybe better https://github.com/aws-amplify/amplify-backend/blob/01ebbc497db483afd3fe0a0d85d1b00a0780171d/packages/cli/src/commands/generate/generate_command_factory.test.ts#L34-L40